### PR TITLE
Openlineage support - Add Extractor for `ExportFileOperator`

### LINKED ISSUE #903: Openlineage support - Add Extractor for `ExportFileOperator`
**Please describe the feature you'd like to see**
We should be able to extract open lineage info from the `ExportFileOperator`.

**Describe the solution you'd like**
- Add a method `get_openlineage_facets` on the Operator (based on [this doc](https://docs.google.com/document/d/1vPsvHejQ24xTbzpz_LYSf0_ixk9oUuBiEUHVEaF9J2U/edit?usp=sharing))
- The "PythonSDKExtractor" built-in https://github.com/astronomer/astro-sdk/issues/898 should be able to work with `ExportFileOperator.get_openlineage_facets`. Test it to make sure it works

**Acceptance Criteria**

- [ ] Post the screenshot of how it looks in the Openlineage/Marquez UI
- [ ] All checks and tests in the CI should pass
- [ ] Unit tests (90% code coverage or more, [once available](https://github.com/astronomer/astro-sdk/issues/191))
- [ ] Integration tests (if the feature relates to a new database or external service)
- [ ] Docstrings in [reStructuredText](https://peps.python.org/pep-0287/) for each of  methods, classes, functions and module-level attributes (including Example DAG on how it should be used)
- [ ] Exception handling in case of errors
- [ ] Logging (are we exposing useful information to the user? e.g. source and destination)
- [ ] Improve the documentation (README, Sphinx, and any other relevant)

### PR DESCRIPTION
**Pull request by rajaths010494**
**Please describe the feature you'd like to see**
We should be able to extract open lineage info from the ExportFileOperator.

**Describe the solution you'd like**

- Add a method get_openlineage_facets on the Operator (based on [this doc](https://docs.google.com/document/d/1vPsvHejQ24xTbzpz_LYSf0_ixk9oUuBiEUHVEaF9J2U/edit?usp=sharing))
- The "PythonSDKExtractor" built-in [#898](https://github.com/astronomer/astro-sdk/issues/898) should be able to work with ExportFileOperator.get_openlineage_facets. Test it to make sure it works

**Acceptance Criteria**

- [x] Post the screenshot of how it looks in the Openlineage/Marquez UI
- [x]  All checks and tests in the CI should pass
- [x]  Unit tests (90% code coverage or more, [once available](https://github.com/astronomer/astro-sdk/issues/191))
- [x]  Integration tests (if the feature relates to a new database or external service)
- [x] Docstrings in [reStructuredText](https://peps.python.org/pep-0287/) for each of methods, classes, functions and module-level attributes (including Example DAG on how it should be used)
- [x] Exception handling in case of errors
- [x]  Logging (are we exposing useful information to the user? e.g. source and destination)
- [x]  Improve the documentation (README, Sphinx, and any other relevant)

closes #903

### CODE REVIEW

File: python-sdk/src/astro/sql/operators/export_file.py [hunk 1]
```diff
@@ -62,6 +73,71 @@ def execute(self, context: Context) -> File:
         else:
             raise FileExistsError(f"{self.output_file.path} file already exists.")
 
+    def get_openlineage_facets(self, task_instance) -> OpenLineageFacets:  # skipcq: PYL-W0613
+        """
+        Collect the input, output, job and run facets for export file operator
+        """
+        input_dataset: list[OpenlineageDataset] = []
+        if isinstance(self.input_data, BaseTable):
```
Comment (pankajastro) [inline python-sdk/src/astro/sql/operators/export_file.py:81]: @utkarsharma2 do we need check for temp table config here https://github.com/astronomer/astro-sdk/pull/1144/files#r1012835723

Comment (rajaths010494) [reply python-sdk/src/astro/sql/operators/export_file.py]: added condition for temp table

**Comment by pankajastro** [comment]
@rajaths010494 can you please add the same screenshot here too https://github.com/astronomer/astro-sdk/pull/1144#issuecomment-1293523419

**Review by pankajastro — APPROVED** [review]
LGTM

**Pull request merged**